frog/imports/client/Wiki/ModalSettings.js

```diff
@@ -0,0 +1,142 @@
+// @flow
+
+import React, { useState } from 'react';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import TextField from '@material-ui/core/TextField';
+import Switch from '@material-ui/core/Switch';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Divider from '@material-ui/core/Divider';
+import { withStyles } from '@material-ui/styles';
+
+import { Modal } from './components/Modal';
+import { type WikiSettingsT } from './index';
+
+type ModalSettingsPropsT = {
+  callback: WikiSettingsT => void,
+  hideModal: () => void,
+  currentSettings: WikiSettingsT,
+  classes: Object
+};
+
+const styles = {
+  group: {
+    margin: '1vh'
+  },
+  divider: {
+    margin: '2vh'
+  },
+  label: {
+    paddingTop: '2vh'
+  },
+  helperText: {
+    display: 'inline',
+    paddingLeft: '1vw',
+    fontSize: '1rem',
+    letterSpacing: '0.001em'
```

Comment:
is this necessary?

Reply:
Removed `letter-spacing` but the other ones are necessary.